core/common/src/main/java/alluxio/util/UnderFileSystemUtils.java

```diff
@@ -158,5 +160,14 @@ public static boolean isSwift(UnderFileSystem ufs) {
     return "swift".equals(ufs.getUnderFSType());
   }
 
+  /**
+   * @param uri the UFS path
+   * @return the bucket or container name of the object storage
+   */
+  public static String getBucketName(AlluxioURI uri) {
+    Preconditions.checkState(isObjectStorage(uri.toString()));
+    return uri.getAuthority();
```

Comment:
This seems strange to me, are we saying for a URI like `s3a://my_bucket/file`, `my_bucket` is actually the authority?